### ISSUE TITLE
VPC always created in us-east-1

### ISSUE THREAD

**Issue by gabeng** [chapmanb/bcbio-nextgen-vm#155]
No matter which AWS region I put into `~/.bcbio/elasticluster/config`, the VPC is always created in `us-east-1` by `bcbio_vm.py aws vpc`. How can I change that? If this is not set in the elasticluster config - where is it set?

**Comment by chapmanb** [comment]
Ben;
Sorry about the confusion, I know this setup is not straightforward to adjust for non us-east runs. These setup commands are bcbio specific and happen prior to running elasticluster, so use standard boto/AWS arguments to determine the region. You should be able to adjust the region by exporting this to your desired region:
```
export AWS_DEFAULT_REGION=us-east-1
```
or the other AWS methods of setting default regions:

http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html

Hope this gets the VPC created cleanly for you in the region you want.

**Comment by tomsing1** [comment]
I have encountered the same problem on Mac OS X 10.12.1, trying to create a VPC in us-west-2. 

As advised above I executed 
`export AWS_DEFAULT_REGION=us-west-2`
before running 
`bcbio_vm.py aws vpc`
but the VPC kept popping up in `us-east-1`.

For good measure, I also changed all `east-1` into `west-2` in `~/.bcbio/elasticluster/config`, but that didn't help, either.

Any ideas what's going on?

I am running the following versions:
```
bcbio-nextgen             1.0.1a0                  py27_0    bioconda
bcbio-nextgen-vm          0.1.0a                  py27_86    bioconda
```
